### DISCUSSION TITLE
Shortlist Of 'Awesome' Candidates

### DISCUSSION THREAD

**Discussion by will-iamalpine** [Green-Software-Foundation/awesome-green-software#29] [Ideas]
Tools to evaluate and consider adding:

EcoIndex.fr
PowerAPI

@dtoakley @navveenb

**Comment by mueller-fr** [comment]
I found one, too!
[The Greenweb Foundation's directory for green web hosting services](https://app.greenweb.org/directory/)!

I searched for `greenweb.org` and did not find it in the `readme.md`. So I guess, we can still add it.

Update: there are links to the parent page `thegreenwebfoundation.org` and other subpages. The directory mentioned above, where you can set your filters, is not listed, yet.
